src/stactools/naip/stac.py

```diff
@@ -121,30 +123,45 @@ def create_item(
         )
 
     if fgdc_metadata_href is not None:
-        fgdc_metadata_text = read_text(fgdc_metadata_href)
-        fgdc = parse_fgdc_metadata(fgdc_metadata_text)
-    else:
-        fgdc = {}
+        if int(year) == 2020:
+            file_xpath = """gmd:identificationInfo/gmd:MD_DataIdentification/gmd:citation/
+            gmd:CI_Citation/gmd:title/gco:CharacterString"""
+
+            with fsspec.open(fgdc_metadata_href) as file:
+                root = XmlElement(
+                    etree.parse(file, base_url=fgdc_metadata_href).getroot()
+                )
+                resource_desc = root.find_text_or_throw(
+                    "".join(file_xpath.split()), missing_element("File Identifier")
```

Comment:
I don't understand what this is doing. There's a string above with no spaces in it, and it's being split and rejoined with no space?